plugins/modules/junos_lldp_interface.py

```diff
@@ -54,6 +54,8 @@
 extends_documentation_fragment:
 - junipernetworks.junos.junos
 
+
```

Comment:
Is the space required?

Reply:
Looks like something up with the script, anyway I am going to remove similar extra lines from all these modules.